### ISSUE TITLE
Python Notebook kernel crashes on cell 2 in sample.ipynb

### ISSUE THREAD

**Issue by rcabatic** [kvos/CoastSat#262]
I have been following the instructions to test out the sample.ipynb on Anaconda3. 

Everything is great until I get to the second cell: 

```
# region of interest (longitude, latitude)
polygon = [[[151.2957545, -33.7012561],
            [151.297557, -33.7388075],
            [151.312234, -33.7390216],
            [151.311204, -33.701399],
            [151.2957545, -33.7012561]]] 
# it's recommended to convert the polygon to the smallest rectangle (sides parallel to coordinate axes)       
polygon = SDS_tools.smallest_rectangle(polygon)
```
on the `polygon = SDS_tools.smallest_rectangle(polygon)` line, my kernel dies and restarts. 

I have tried to delete and redownload the CoastSat-Master file, but no luck.
Any suggestions on how I could proceed are appreciated! 

Other specs: 
- I'm running on a Mac (2015).
- Files kept on an external hard-drive

**Comment by rcabatic** [comment]
The cell works fine if I just comment out that line, but still curious as to how I can fix problem with SDS_tools.smallest_rectangle

**Comment by kvos** [comment]
I don't know what could be causing the kernel to crash, without an error output or log file it's hard to understand what's going on. Have you tried with different coordinates (like the example at Narrabeen) and it still does that?

**Comment by rcabatic** [comment]
Closed because I got it to work!( Although it was not in the way I thought.)

I will outline it below for those who may run into this as well.

I am running on Mac OS Big Sur. When I run the first cell in the example notebook, I was getting the error _"OSError: Could not find lib c or load any of its variants []."_ which was traced back to the Shapely package.

Two other colleagues who were running the example notebook on Mac OS also go this error. 

After one of them pointed me to [this thread](https://github.com/conda-forge/shapely-feedstock/issues/53), and I did what one of the comments (Guochaoer) said, the notebook now works all the way through and I do not run into any issues. 

Basically, you must access the geos.py file (i.e - .../users/richellecabatic/anaconda3/envs/GIS/lib/python3.7/site-packages/shapely/geos.py").
Then, manually change line 113 from `free = load_dll('c').free to`:
`free = load_dll('c', fallbacks=alt_paths).free`.
